zerver/migrations/0615_merge_20241017_1404.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 5.0.9 on 2024-10-17 14:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0610_system_bot_avatars"),
+        ("zerver", "0614_remove_realm_move_messages_between_streams_policy"),
+    ]
+
+    operations = []
```

Comment:
You almost never want to make a merge migration; see https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html.

Reply:
Noted